mix.exs

```diff
@@ -69,6 +70,7 @@ defmodule Tesla.Mixfile do
       # other
       {:fuse, "~> 2.4", optional: true},
       {:telemetry, "~> 0.3", optional: true},
+      {:cowlib, "~> 1.0.2", override: true, optional: true},
```

Comment:
Why is this needed?

Reply:
gun requires {:cowlib, "~> 2.6.0"}, that's why `override: true` was added

Reply:
removed after update `httparrot` dependency